interop/tapir/src/main/scala/caliban/interop/tapir/WebSocketHooks.scala

```diff
@@ -9,11 +9,12 @@ trait StreamTransformer[-R, +E] {
 }
 
 trait WebSocketHooks[-R, +E] { self =>
-  def beforeInit: Option[InputValue => ZIO[R, E, Any]] = None
-  def afterInit: Option[ZIO[R, E, Any]]                = None
-  def onMessage: Option[StreamTransformer[R, E]]       = None
-  def onPong: Option[InputValue => ZIO[R, E, Any]]     = None
-  def onAck: Option[ZIO[R, E, ResponseValue]]          = None
+  def beforeInit: Option[InputValue => ZIO[R, E, Any]]     = None
+  def afterInit: Option[ZIO[R, E, Any]]                    = None
+  def onMessage: Option[StreamTransformer[R, E]]           = None
+  def onPong: Option[InputValue => ZIO[R, E, Any]]         = None
+  def onPing: Option[Option[InputValue] => ZIO[R, E, Any]] = None
```

Comment:
Should we make this `Option[Option[InputValue] => ZIO[R, E, Option[ResponseValue]]]`? 

Spec shows that pong can have a payload as well.

https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md#pong

Reply:
Right, makes sense to me, added. :+1: